app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2> <%= @person[:name] %>'s Tasks are: </h2>
```

Comment:
Watch for consistency in how you are using your model fields. Recommend `model.field` rather than `model[:field]`.